### ISSUE TITLE
Too big power consumption and heating while making voice call

### ISSUE THREAD

**Issue by ioctl-user** [signalapp/Signal-Android#9687]
- [X] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [X] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Phone heats and consumes too much power 

### Steps to reproduce
- Do not turn on option "Translate calls through server". I.e. use direct phone-to-phone connection 
- Use mobile connection (I have not tried with WiFi)
- Make voice call

**Actual result:** Phone connected to original power adapter with 10..15 Watt output loose battery charge with about 1 Ampere current. Device temperature reach 48 *C. 

### Device info
Phone with high-end ARMv8 CPU from 2018 year.
**Android version:** LineageOS 17.1 (Android 10) without GCM.
**Signal version:** 4.59.10

**Comment by Lyndeno** [comment]
I have been experiencing this as well. Extreme battery usage on a voice call (possibly video as well). Screen is left off.

OnePlus 7 Pro
Oxygen OS 10.3.3 (Android 10)
Signal 4.62.1

**Comment by ioctl-user** [comment]
Today I updated app to the latest version and measured power consumption without power supply during voice call. It was 1.2 Ampere and phone was very hot.

Here is a debug log: https://debuglogs.org/7d49eb786f6de66b5bc49a3d54e5a04ff8f602d42ac718b32835377a47734bf5

Phone call was started about 2020-06-09 19:00:00 by local time.

**Comment by ioctl-user** [comment]
Signal 4.70.5 on the same phone: power consumption during voice call ~700 mA.

It's definitely better, but still too big.

**Comment by ohimu** [comment]
Probably same bug as #11067 #11294

**Comment by LightSoar** [comment]
Overheating on video calls within ~20 min on:
- Android 10 @ Samsung Galaxy S20 FE
- MIUI 11 @ Xiaomi Redmi Note 8

No overheating on:
- Android 10 @ Nokia 6.1

**Comment by ioctl-user** [comment]
> Is this still relevant?

Yes, it's still present.

**Comment by MrWaste** [comment]
Check this post just updated it again. https://github.com/signalapp/Signal-Android/issues/9798

**Comment by ferdinandschober** [comment]
Any updates? This is a big problem for me.

**Comment by cody-signal** [comment]
No updates. It's something we're aware of, and bring up with the calling team, but as far as I'm aware there's no current plans to address this directly. They are constantly looking at ways to improve calling so hopefully things improve solely because of that.

**Comment by ferdinandschober** [comment]
Yeah I commented mainly so this issue would not get closed automatically

**Comment by cody-signal** [comment]
![image](https://user-images.githubusercontent.com/66265416/166240179-baf624fe-4d2c-48aa-86e6-c46a2bb8cd9d.png)

**Comment by iaxat** [comment]
This issue still persists. Is there any update?
